src/plugins/workspace/public/components/workspace_creator/workspace_creator_form.tsx

```diff
@@ -138,28 +148,47 @@ export const WorkspaceCreatorForm = (props: WorkspaceCreatorFormProps) => {
             </>
           )}
           {permissionEnabled && (
-            <>
-              <EuiTitle
-                {...generateRightSidebarScrollProps(RightSidebarScrollField.Member)}
-                size="s"
-              >
-                <h3>{usersAndPermissionsCreatePageTitle}</h3>
-              </EuiTitle>
-              <EuiText size="xs">
-                {i18n.translate('workspace.creator.form.usersAndPermissionsDescription', {
-                  defaultMessage:
-                    'You will be added as an owner to the workspace. Select additional users and user groups as workspace collaborators with different access levels.',
-                })}
-              </EuiText>
-              <EuiSpacer size="m" />
-              <WorkspacePermissionSettingPanel
-                errors={formErrors.permissionSettings?.fields}
-                onChange={setPermissionSettings}
-                permissionSettings={formData.permissionSettings}
-                disabledUserOrGroupInputIds={disabledUserOrGroupInputIdsRef.current}
-                data-test-subj={`workspaceForm-permissionSettingPanel`}
-              />
-            </>
+            <EuiSplitPanel.Outer direction="column">
+              <EuiSplitPanel.Inner>
+                <EuiTitle
```

Comment:
Can we use `EuiTitle + h2` as panel title [like this](https://github.com/opensearch-project/OpenSearch-Dashboards/blob/main/src/plugins/workspace/public/components/workspace_creator/creator_details_panel.tsx#L42-L48) to align with the guidance.

Reply:
ok